### PR TITLE
Updated README with package badges linked to install page (NuGet / MyGet)

### LINKED ISSUE #333: Add link to nuget
>At present the code is pre-release but when ready it will be available on Nuget.

After all, there are https://www.nuget.org/packages/SixLabors.ImageSharp/1.0.0-beta0001

### PR DESCRIPTION
**Pull request by ryanelian**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

I think it is important to provide links pointed to respective NuGet / MyGet package pages in README, so people can discover the project more easily.

badges

These high resolution badges should be auto-updating to reflect the latest and greatest package versions.

Resolves #333

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +50,9 @@ Packages include:
   - Various vector drawing methods for drawing paths, polygons etc.
   - Text drawing.
 
+- **SixLabors.ImageSharp.Web**
```
Comment (tocsoft) [inline README.md]: (as above)
ImageSharp.Web is in a separate [respository](https://github.com/SixLabors/ImageSharp.Web) so shouldn't really be included in this readme

File: README.md [hunk 2]
```diff
@@ -27,11 +27,12 @@ Built against .Net Standard 1.1 ImageSharp can be used in device, cloud, and emb
...
+| Package Name                   | Release (NuGet) | Nightly (MyGet) |
+|--------------------------------|-----------------|-----------------|
+| `SixLabors.ImageSharp`         | [![NuGet](https://img.shields.io/nuget/v/SixLabors.ImageSharp.svg)](https://www.nuget.org/packages/SixLabors.ImageSharp/) | [![MyGet](https://img.shields.io/myget/sixlabors/v/SixLabors.ImageSharp.svg)](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp) |
+| `SixLabors.ImageSharp.Drawing` | [![NuGet](https://img.shields.io/nuget/v/SixLabors.ImageSharp.Drawing.svg)](https://www.nuget.org/packages/SixLabors.ImageSharp.Drawing/) | [![MyGet](https://img.shields.io/myget/sixlabors/v/SixLabors.ImageSharp.Drawing.svg)](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp.Drawing) |
+| `SixLabors.ImageSharp.Web`     | [![NuGet](https://img.shields.io/nuget/v/SixLabors.ImageSharp.Web.svg)](https://www.nuget.org/packages/SixLabors.ImageSharp.Web/) | [![MyGet](https://img.shields.io/myget/sixlabors/v/SixLabors.ImageSharp.Web.svg)](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp.Web) |
```
Comment (tocsoft) [inline README.md]: ImageSharp.Web is actually in a separate [respository](https://github.com/SixLabors/ImageSharp.Web) so shouldn't really be included in this readme.

**Comment by ryanelian** [comment]
oh lmao.
sorry, I didn't know.
ok, there you go.

**Comment by tocsoft** [comment]
Thanks for that.

FYI
For a short time it did live in this repo as a branch but we split it out prior to beta as it was more of a package that used the core imagesharp that a core piece by itself.

**Pull request merged**